README.md

```diff
@@ -1,2 +1,30 @@
 # grpc-in-go
-Build microservices communicating thourgh gRPC in Golang
+
+Build a mood tracker using Go and gRPC.
+
+## Prerequisites
+
+Install [Go](https://go.dev/doc/install) 1.17
+
+Install protoc dependencies
```

Comment:
Tell me why, I want to know what I'm adding to my computer.
Also, can you have the CI generate the files ?

Reply:
The goal is to make a tutorial and to understand and learn at each step. 
Even though I can add it to the CI in a next step.